pipeline_dp/aggregate_params.py

```diff
@@ -650,18 +650,44 @@ class AddDPNoiseParams:
     sensitivity by contribution bounding and relies on the caller to ensure the
     provided data satisfies the provided bound.
 
-   Attributes:
+    The noise in DP is scaled with sensitivity, where sensitivity for query q is
+     sensitivity = \max_{D, D' neighbouring} ||q(D)-q(D')||
+    Where ||.|| should be a proper norm, L1 for Laplace Mechanism, L2 for
+    Gaussian Mechanism.
+    See https://en.wikipedia.org/wiki/Norm_(mathematics)#p-norm on norm details.
+
+    In our case, q values are represented as collection of
+    (partition_key, value). Where partition_key is any identifier, value is
+    a scalar. q values can be represented as a vector indexed by partition_key.
+    One of the convenient ways to define l1/l2 sensitivity is through l0/linf
+    sensitivities.
+    Namely
+
+    l1_sensitivity = l0_sensitivity*linf_sensitivity
+    l2_sensitivity = sqrt(l0_sensitivity)*linf_sensitivity
+
+    For applying the Laplace mechanism or l1_sensitivity or
+    l0_sensitivity, linf_sensitivity must be specified.
```

Comment:
optional: maybe put l0 linf into brackets () to highlight that both of them have to specified